### DISCUSSION TITLE
Custom Cards

### DISCUSSION THREAD

**Discussion by Kaiserdragon2** [zixpo/candybar#150] [Q&A]
Is there a way to add custom cards to the dashboard in the home tab?

**Comment by Donnnno** [comment]
@moertel you did this with the "Quick Help" option, right?

**Reply by moertel** [reply]
Oh yes, true, I "hacked" one of the existing cards (the "Donation" one) and replaced all the translations and icons to make it look like a custom one. This only works with the "Title, summary, link" scheme, though.

**Reply by moertel** [reply]
@Kaiserdragon2 is there a specific thing you'd like to accomplish? If you'd be happy with something like in the screenshots below, I can help guide you through it without any code changes in CandyBar.

![Screenshot_20230719_214515_Retro Mode - Neon.jpg](https://github.com/zixpo/candybar/assets/1823966/e629870c-c32c-4f5b-834a-03d1ee320d65)

![Screenshot_20230719_214519_Retro Mode - Neon.jpg](https://github.com/zixpo/candybar/assets/1823966/23844de8-f5cc-4859-a9ca-bddd85649de3)

**Reply by Kaiserdragon2** [reply]
I want to show an option to add an shortcut to the homescreen. With the old API the shortcut would be shown in the Widgets menu, but with the new API i need to evoke  the action from somewhere else.

**Reply by Kaiserdragon2** [reply]
I would be happy with a quick guide to archive something similar.
